### DISCUSSION TITLE
Can we get Scrolling Chat in MP?

### DISCUSSION THREAD

**Discussion by Trippeh** [OpenRCT2/OpenRCT2#15702] [Feature requests]
- [x] Multiplayer?

**Comment by AaronVanGeffen** [comment]
I thought this had already been done, but both #3034 and #4293 have stranded, meaning this still needs doing.

**Comment by RayKoopa** [comment]
This would be a very nice feature still!

Right now, the chat is still very statically implemented with custom drawing and not being an actual window, so adding scroll functionality with a proper scrollbar seems (at least to me) not too trivial.
Otherwise maybe turning the chat into a real window to reuse some widget functionality might also allow it to become resizable, but I'm too unexperienced to judge the pros and cons of that.
